docker/pool/build-screen/bower.json

```diff
@@ -2,6 +2,6 @@
   "name": "build-screen",
   "private": true,
   "dependencies": {
-    "bootstrap": ">=3.3.0"
+    "bootstrap": "=3.3.4"
```

Comment:
I agree with you to use newer version of libraries as some features/fixes are merged, so what do you think to use `>=` version semantics?
If problem you met is came from any version except for `3.3.4`, I agree to use fixed version.